docs/docs/query-behind-the-scenes.md

```diff
@@ -2,9 +2,9 @@
 title: How Queries Work
 ---
 
-We're talking about GraphQL queries here. These can be tagged graphql expressions at the bottom of your component source file (e.g. [query for Gatsby frontpage](https://github.com/gatsbyjs/gatsby/blob/master/www/src/pages/index.js#L165)), StaticQueries within your components (e.g. [showcase site details](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/showcase-details.js#L103)), or fragments created by plugins (e.g. [gatsby-source-contentful](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-contentful/src/fragments.js)).
+You're talking about GraphQL queries here. These can be tagged graphql expressions at the bottom of your component source file (e.g. [query for Gatsby frontpage](https://github.com/gatsbyjs/gatsby/blob/master/www/src/pages/index.js#L165)), StaticQueries within your components (e.g. [showcase site details](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/showcase-details.js#L103)), or fragments created by plugins (e.g. [gatsby-source-contentful](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-contentful/src/fragments.js)).
```

Comment:
This reads weird. Maybe we need an exception in the style guide? 🤷‍♂ 

@shannonbux @marcysutton

Reply:
It's somewhat subjective how to structure sentences, but I agree that this doesn't quite work here (the language used was odd to begin with). I'd recommend restructuring this and the following paragraphs to not talk about "talking", but rather present the information more directly. In some cases "we" means Gatsby's code, so it would be more appropriate to talk about the code as a subject since it's doing the action (not the user/developer).

Here's a recommendation:

```suggestion
As part of Gatsby's data layer, GraphQL queries can be specified as tagged graphql expressions at the bottom of your component source file (e.g. [query for Gatsby frontpage](https://github.com/gatsbyjs/gatsby/blob/master/www/src/pages/index.js#L165)), StaticQueries within your components (e.g. [showcase site details](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/showcase-details.js#L103)), or fragments created by plugins (e.g. [gatsby-source-contentful](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-contentful/src/fragments.js)).
```

Reply:
oh yes, I can see how it would be easy to make this mistake since the style guide says "change 'we're' to 'you're.'" This is a good example of when it's not necessary to mention either we or you at all; it's more accurate and efficient to just talk about Gatsby.